src/test/java/com/crud/tasks/mapper/TrelloMapperTestSuite.java

```diff
@@ -71,33 +71,31 @@ public void mapToList() {
     }
 
     @Test
-    public void mapToBoards() {
-        //Given
+    public void testMapToBoardsMethod() {
+        //When
```

Comment:
http://naukaoduszy.pl/kolorowy-test/ mniej wiecej to samo osiagasz stosujac nazwe expected jako wejsce